powermon/device.py

```diff
@@ -90,10 +97,12 @@ def runLoop(self, force=False):
             for command in self.commands:
                 if force or command.dueToRun():
                     # run command
-                    result = self.port.run_command(command)
+                    result: Result = self.port.run_command(command)
                     # decode result
-                    self.port.protocol.decode(result)
+                    self.port.get_protocol().decode(result)
```

Comment:
whats the logic behind the change to get_protocol()?

Reply:
By switching from direct variable access to a getter method we can add a return type. I added a python linter to my IDE and it's telling me all the python best practices which I am incorporating.

Reply:
> By switching from direct variable access to a getter method we can add a return type. I added a python linter to my IDE and it's telling me all the python best practices which I am incorporating.

Which linter? (and what IDE are you using?) - coz seems like sensible recommendations and would make sense for me to use it too

Reply:
I'm using visual studio code with pylint